htcrystalball/display.py

```diff
@@ -67,10 +67,8 @@ def results(result: dict, verbose: bool, matlab: bool,
     # write table and wall-time info to console
     if verbose:
         console.print(table)
-        console.print("USAGE:")
-        console.print("[green]--- [/green]<= 95%")
-        console.print("[yellow]--- [/yellow]> 95%")
-        console.print("[red]--- [/red]> 100%")
+        console.print("LEGEND:")
+        console.print("[green]█[/green] (<= 95%); [yellow]█[/yellow] (95-100%); [red]█[/red] (> 100%)")
```

Comment:
Never heard of the "block" glyph before. That's very neat. :-)

Reply:
Yep, it's quite useful. It's the one you see at the command prompt, when your cursor is blinking. :-)